src/components/reference/_language-extensions-tags.mdx

```diff
@@ -5,35 +5,53 @@ or by default) uncomment the following two lines and closing tag below the table
-| Scala      | `.scala`, `.sc`        | `scala`                              |
-| Solidity   | `.sol`                 | `solidity`                           |
-| Terraform  | `.tf`                  | `hcl`                                |
-| TypeScript | `.ts`, `.tsx`          | `ts`, `tsx`, `typescript`            |
-| TSX        | `.ts`, `.tsx`          | `ts`, `tsx`, `typescript`            |
-| YAML       | `.yaml`                | `yaml`                               |
+The following table includes languages that Semgrep supports, their file extension, and valid values that Semgrep rules require in the `language` or `languages` key.
+
+| Language                            | Extensions                           | `language` and `languages` key values |
+|:------------------------------------|:-------------------------------------|:--------------------------------------|
+| Apex (only in Semgrep Pro Engine)   | `.cls`                               | `apex`                                | 
+| Bash                                | `.bash`, `.sh`                       | `bash`, `sh`                          |
+| C                                   | `.c`                                 | `c`                                   |
+| Clojure                             | `.clj`, `.cljs`, `.cljc`, `.edn`     | `clojure`                             |
+| C++                                 | `.cc`, `.cpp`                        | `cpp`, `c++`                          |
+| C#                                  | `.cs`                                | `csharp`, `c#`                        |
+| Dart                                | `.dart`                              | `dart`                                |
+| Dockerfile                          | `.dockerfile`, `.Dockerfile`         | `dockerfile`, `docker`                |
+| Elixir                              | `.ex`, `.exs`                        | `ex`, `elixir`                        |
+| Generic                             |                                      | `generic`                             |
+| Go                                  | `.go`                                | `go`, `golang`                        |
+| HTML                                | `.htm`, `.html`                      | `html`                                |
+| Java                                | `.java`                              | `java`                                |
+| JavaScript                          | `.js`, `.jsx`                        | `js`, `javascript`                    |
+| JSON                                | `.json`, `.ipynb`                    | `json`                                |
+| Jsonnet                             | `.jsonnet`, `.libsonnet`             | `jsonnet`                             |
+| JSX                                 | `.js`, `.jsx`                        | `js`, `javascript`                    |
```

Comment:
I included JSX as a separate language here, although this is not a separate language according to https://github.com/returntocorp/semgrep-langs/blob/main/lang.json we list it separately in [Semgrep OSS Engine](https://semgrep.dev/docs/supported-languages/#semgrep-oss-engine) supported languages table.